docs/geedocs/answer/4485225.html

```diff
@@ -490,8 +392,8 @@ <h2 id="command_line_reference">Command line reference</h2>
   <h3><a id="gerewritedbroot" name="gerewritedbroot"></a>gerewritedbroot</h3>
   <h4>Usage</h4>
   <code><strong>gerewritedbroot</strong></code> <code>--source=<em>server_name</em> --icon_directory=<em>directory</em><br/>
-    --dbroot_file=<em>filename</em> [--search_server=<em>server_name</em>]<br/>
-    [--search_port=<em>num]</em> [--kml_map_file=<em>filename]</em> [--kml_server=<em>server_name]</em><br/>
+    --dbroot_file=<em>filename</em>  --kml_map_file=<em>filename</em><br/>
+    [--search_service=<em>search_service_url</em>] [--kml_server=<em>server_name]</em><br/>
```

Comment:
Yeah, at least, according to what I have on my machine. `server` and `port` became `service` URL.